doc/pod_chaos.md

```diff
@@ -39,6 +40,7 @@ Description:
 
 * **action** defines the specific chaos action for the pod. In this case, it is pod failure.
 * **mode** defines the mode to run chaos action. Supported mode: `one / all / fixed / fixed-percent / random-max-percent`.
+* **value** depends on the value of `mode`. If `mode` is `one` or `all`, leave `value` empty. If `fixed`, provide an integer of pods to do chaos action. If `fixed-percent`, provide a number from 0-100 to specify the max percent of pods the server can do chaos action. If `random-max-percent`, provide a number from 0-100 to specify the percent of pods to do chaos action
```

Comment:
If `fixed-percent`, provide a number from 0-100 to specify the percent of pods the server can do chaos action. If `random-max-percent`, provide a number from 0-100 to specify the max percent of pods to do chaos action.   

In addition, Can you help to update PodChaos field comment?

Reply:
The change has been made.